vuepress/docs/next/docs/consume/operator-intro.md

```diff
@@ -35,7 +35,7 @@ When choosing how to deploy your operator there are no right or wrong answers. T
 - When teams are small and self managing from an operational perspective a namespace scoped deployment is a simpler architecture
```

Comment:
When teams are small and self managing from an operational perspective a namespace scoped deployment is a simpler architecture -> When teams are small and self-managing from an operational perspective, a namespace-scoped deployment is a simpler architecture